Sources/MapboxMaps/Style/Types/Expression.swift

```diff
@@ -245,6 +245,13 @@ public struct Expression: Codable, CustomStringConvertible, Equatable {
 
             if let validArray = try? container.decode([Double].self) {
                 self = .array(validArray)
+                return
+            }
+
+            // Attempt to decode an "empty" json if all else fails
```

Comment:
This comment seems like that in decoding, an empty json will be the fail all case. 

I don't think a comment is needed. Or maybe say that this is to account for empty json's and provide an example.